_docs/contributing.md

```diff
@@ -31,14 +31,17 @@ test suite which covers a variety of scenarios. Changes made to the code should
 not break existing features. We strongly recommend running the standard tests
 before you send a pull request to development. 
```

Comment:
what does 'standard tests' mean? this could mean different things to different people.

Reply:
I see the problem. "Standard" removed in 3785e2e195f9c6fecab3880e69f32084d89d9ec8.